### DISCUSSION TITLE
My First AppCenter App

### DISCUSSION THREAD

**Discussion by techdev5521** [elementary/edw#19] [Accepted Talks]
I'm currently a web developer with experience in React and Python. I'm in the process of building simple apps for elementary to get familiar with the workflow and tooling. The timing of this conference puts me in a spot to record my path and present a condensed version of getting started to new commers.

**Comment by jalcine** [comment]
This would be dope as I'm investigating how to build social applications for Elementary

**Reply by techdev5521** [reply]
Can you say more about what you mean by "social applications"?

**Reply by jalcine** [reply]
Like first-class clients for social protocols like ActivityPub and other ones provided by the IndieWeb!

**Comment by techdev5521** [comment]
As a summary/clarification to the proposed idea:

In this talk I'll go over creating my first elementaryOS application and my day to day usage of elementaryOS as a visually impaired developer. I'll talk about the process of writing, submitting and maintaining an application, the tooling for the process and the interaction with the rest of the system.

**Comment by cassidyjames** [comment]
@techdev5521 we've accepted your talk, including your summary/clarification. :) You can get started writing it now if you'd like, and we will provide further instructions/guidance around the recording once we've accepted all the talks.

**Reply by techdev5521** [reply]
Lezgoooo! 🕺
